django/db/models/options.py

```diff
@@ -467,6 +467,17 @@ def concrete_fields(self):
         )
 
     @cached_property
+    def concrete_fields_attnames_enumerated(self):
```

Comment:
You'll want to make sure this is in `FORWARD_PROPERTIES` so `_expire_cache` clears it up.

Reply:
I'm not sure if this should be tested and how to implement the test.